templates/invite.hamlet

```diff
@@ -1,4 +1,6 @@
 $maybe link <- maybe_link
+    <h1 .text-center>
+        Invite
```

Comment:
Now that I'm thinking about it, "Invite" is a pretty meaningless title. It should probably actually be a meaningful title like "Invite Users for New Roles for #{projectName project}" or something like that…